timm/models/vgg.py

```diff
@@ -38,8 +38,8 @@ def __init__(
             kernel_size=7,
             mlp_ratio=1.0,
             drop_rate: float = 0.2,
-            act_layer: nn.Module = None,
-            conv_layer: nn.Module = None,
+            act_layer: Optional[Type[nn.Module]] = None,
+            conv_layer: Optional[Type[nn.Module]] = None,
```

Comment:
There is an existing bug here. `act_layer` and `conv_layer` both default to None, but None is an invalid input and will cause the code to crash. Should we change the default and make it non-optional?